### PR TITLE
feat: website design

### PR DESCRIPTION
**Pull request by moripen**

### CODE REVIEW

**Comment by cobraz** [comment]
@moripen looking at the previews, there are some design issues. Should I only look at the code for now, or the design as well?

File: components/layout/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { Flex } from '@chakra-ui/react';
+import { SEO } from '../seo';
+import { Footer } from './footer/footer';
+import { Header } from './header/header';
+
+type Props = {
+  children?: ReactNode;
+};
+
+export const Layout = ({ children }: Props) => (
```
Comment (braaar) [inline components/layout/layout.tsx:11]:
```suggestion
export const Layout : React.FC<PropsWithChildren> = ({ children }) => (
```

File: components/layout/page-section.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export interface PageSectionProps {
+  backgroundColor?: 'green';
+  children?: ReactNode;
+}
+
+export const PageSection: React.FC<PageSectionProps> = ({
+  children,
+  backgroundColor: backgroundColorProp,
+}) => {
```
Comment (braaar) [inline components/layout/page-section.tsx:12]:
```suggestion
export interface PageSectionProps {
  backgroundColor?: 'green';
}

export const PageSection: React.FC<PropsWithChildren<PageSectionProps>> = ({
  children,
  backgroundColor: backgroundColorProp,
}) => {
```

File: components/layout/about/contact-person.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    AspectRatio,
+    Box,
+    Flex,
+    Grid,
+    Image,
+    Link,
+    Text,
+  } from '@chakra-ui/react';
+  import NextLink from 'next/link';
+
```
Comment (cobraz) [inline components/layout/about/contact-person.tsx:11]: Can you please run `prettier`?

Comment (cobraz) [reply components/layout/about/contact-person.tsx:11]: … Or maybe leave that for a later PR

File: components/layout/page-section.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren, ReactNode } from 'react';
+import { Box } from '@chakra-ui/react';
+
+export interface PageSectionProps {
+  backgroundColor?: 'green';
+}
+
+export const PageSection: React.FC<PropsWithChildren<PageSectionProps>> = ({
+  children,
+  backgroundColor: backgroundColorProp,
```
Comment (cobraz) [inline components/layout/page-section.tsx:10]: Should pass sx?

**Review by cobraz — APPROVED** [review]

File: components/layout/page-section.tsx (hunk 4 above)
Comment (moripen) [reply components/layout/page-section.tsx:10]: We don't use sx for this as its more of an outline component - we do however use it for the components inside it

**Pull request merged**